[2023-12-10 19:41:56,727] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-04T00:00:00+00:00 [queued]>
[2023-12-10 19:41:56,759] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-04T00:00:00+00:00 [queued]>
[2023-12-10 19:41:56,764] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:41:56,772] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:41:56,789] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:41:56,947] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-05-04 00:00:00+00:00
[2023-12-10 19:41:56,954] {standard_task_runner.py:52} INFO - Started process 2979 to run task
[2023-12-10 19:41:56,976] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-05-04T00:00:00+00:00', '--job-id', '830', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpi170jzzp', '--error-file', '/tmp/tmpprmlzokm']
[2023-12-10 19:41:56,986] {standard_task_runner.py:80} INFO - Job 830: Subtask segunda_tarea
[2023-12-10 19:41:57,364] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-04T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:41:57,658] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-05-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-04T00:00:00+00:00
[2023-12-10 19:41:57,660] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:41:57,663] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:41:57,681] {subprocess.py:85} INFO - Output:
[2023-12-10 19:41:57,688] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:41:57,689] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:41:57,758] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230504T000000, start_date=20231210T194156, end_date=20231210T194157
[2023-12-10 19:41:57,819] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:41:57,935] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
